docs/setup.rst

```diff
@@ -79,6 +79,44 @@ Docker Compose
 Now proceed to `"Setup" <#setup>`__.
 
 
+Heroku
+------
+
+1. Install `Heroku CLI <https://toolbelt.heroku.com/>`_.
+
+2. Create Heroku App::
+
+    $ heroku apps:create <app name>
+
+2. Set application buildpacks::
+
+    $ heroku buildpacks:set heroku/python
+    $ heroku buildpacks:add --index 1 heroku/nodejs
+
+3. Add Postgres addon::
+
+    $ heroku addons:create heroku-postgresql:hobby-dev
+
+4. Add Redis addon::
+
+    $ heroku addons:create heroku-redis:hobby-dev
+
+5. Set environment variables::
+
+    $ heroku config:set REDASH_DATABASE_URL=`heroku config:get DATABASE_URL`
+    $ heroku config:set REDASH_REDIS_URL=`heroku config:get REDIS_URL`
```

Comment:
I'm totally cool with changing the relevant lines in `settings.py` to:

```python
DATABASE_CONFIG = parse_db_url(os.environ.get("REDASH_DATABASE_URL", os.environ.get("DATABASE_URL", "postgresql://postgres")))
```

And:

```python
REDIS_URL = os.environ.get('REDASH_REDIS_URL', os.environ.get('REDIS_URL', "redis://localhost:6379/0"))
```

I can do it myself after merging this pull request, if you prefer.